docs/integrations/destinations/s3.md

```diff
@@ -24,80 +27,85 @@ Prepare S3 bucket that will be used as destination, see [this](https://docs.aws.
 
 NOTE: If the S3 cluster is not configured to use TLS, the connection to Amazon S3 silently reverts to an unencrypted connection. Airbyte recommends all connections be configured to use TLS/SSL as support for AWS's [shared responsibility model](https://aws.amazon.com/compliance/shared-responsibility-model/)
 
-## Step 2: Set up the S3 destination connector in Airbyte
+### Step 2: Set up the S3 destination connector in Airbyte
 
-**For Airbyte Cloud:**
```

Comment:
Hey @arnaudjnn , I think we need to leave these headers in rather than removing them (I would move the env:cloud tag to above this line so that it does show up in Cloud). In the case someone goes to the Docs page outside of the UI (by going directly to https://airbytehq.github.io/integrations/destinations/s3/), we would still want to list both OSS and Cloud on the page as it's the full documentation so they can choose which one to consume (since we won't know which they are using.